src/main/java/org/gridsuite/directory/server/DirectoryController.java

```diff
@@ -65,6 +67,14 @@ public DirectoryController(DirectoryService service) {
         return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(service.listDirectoryContent(directoryUuid));
     }
 
+    @GetMapping(value = "/directories/{directoryUuid}", produces = MediaType.APPLICATION_JSON_VALUE)
+    @ApiOperation(value = "Get directory infos")
+    @ApiResponses(@ApiResponse(code = 200, message = "directory's infos"))
+    public ResponseEntity<Mono<DirectoryAttributes>> getElementInfos(@PathVariable("directoryUuid") String directoryUuid) {
```

Comment:
Why not name the function "getDirectoryInfos" (same name as the method called in service) ?